components/fxa-client/src/fxa_client.udl

```diff
@@ -243,7 +246,18 @@ interface FirefoxAccount {
   // is not desired then the application should discard the persisted account state.
   //
   void disconnect();
-  
+
+  // Log out because of authorization / authentication issues
+  //
+  //
+  // **💾 This method alters the persisted account state.**
+  //
+  // Call this if you know there's an authentication / authorization issue that requires the
+  // user to re-authenticated.  It transitions the user to the [FxaRustAuthState.AuthIssues] state.
+  void logout_from_auth_issues();
```

Comment:
IIUC, we want this to preserve the device id in the case of an authentication issue, so that we don't issue the user a new device id, correct?

What do you think of adding tests for this in the Rust layer to make sure that an authentication after a auth issue correctly uses the old device id?